# This patch fixes a 'cannot assign to a named constant' error that occurs during the build.
# Suggested fix from: https://lists.debian.org/debian-mentors/2014/08/msg00475.html
--- src/62_ctqmc/defs.h.orig	2014-02-15 21:30:20.000000000 +0000
+++ src/62_ctqmc/defs.h	2016-08-22 13:05:09.473620286 +0100
@@ -5,11 +5,11 @@
 
 #define MALLOC(ARR,SIZE)     ABI_ALLOCATE(ARR,SIZE)
 #define FREE(ARR)            ABI_DEALLOCATE(ARR) 
-#define FREEIF(ARR)          IF(ALLOCATED(ARR)) THEN NEWLINE ABI_DEALLOCATE(ARR) NEWLINE END IF
+#define FREEIF(ARR)          IF(ALLOCATED(ARR)) ABI_DEALLOCATE(ARR)
 
 #define DT_MALLOC(ARR,SIZE)  ABI_DATATYPE_ALLOCATE(ARR,SIZE)
 #define DT_FREE(ARR)         ABI_DATATYPE_DEALLOCATE(ARR) 
-#define DT_FREEIF(ARR)       IF(ALLOCATED(ARR)) THEN NEWLINE ABI_DATATYPE_DEALLOCATE(ARR) NEWLINE END IF
+#define DT_FREEIF(ARR)       IF(ALLOCATED(ARR)) ABI_DATATYPE_DEALLOCATE(ARR)
 
 #define myWARNALL(msg)       MSG_WARNING(msg)
 #define myWARN(msg)          call msg_hndl(msg,"WARNING","PERS")
